src/core/modules/Star.hpp

```diff
@@ -283,4 +284,4 @@ struct Star3 {  // 6 byte
 };
 static_assert(sizeof(Star3) == 6, "Size of Star3 must be 6 bytes");
 
-#endif // STAR_HPP
+#endif // STAR_HPP
```

Comment:
You seem to have removed a trailing `\n` char, please place it back.

Reply:
added it back, sorry about that.